src/includes/environment.js

```diff
@@ -750,28 +750,33 @@ pub.bleeds = function(top, right, bottom, left) {
 
 /**
  * @description Inspects a given object or any other data item and prints the result to the console. This is useful for inspecting or debugging any kind of variable or data item. The optional settings object allows to control the function's output. The following parameters can be set in the settings object:
- * <code>showProps<code>: Show or hide properties. Default: <code>true<code>
- * <code>showValues<code>: Show or hide values. Default: <code>true<code>
- * <code>showMethods<code>: Show or hide methods. Default: <code>false<code>
- * <code>maxLevel<code>: Chooses how many levels of properties should be inspected recursively. Default: <code>1<code>
- * <code>propList<code>: Allows to pass an array of property names to show. If propList is not set all properties will be shown. Default: <code>[]<code> (no propList)
+ * <code>showProps</code>: Show or hide properties. Default: <code>true</code>
+ * <code>showValues</code>: Show or hide values. Default: <code>true</code>
+ * <code>showMethods</code>: Show or hide methods. Default: <code>false</code>
+ * <code>maxLevel</code>: Chooses how many levels of properties should be inspected recursively. Default: <code>1</code>
+ * <code>propList</code>: Allows to pass an array of property names to show. If propList is not set all properties will be shown. Default: <code>[]</code> (no propList)
  * If no settings object is set, the default values will be used.
  *
  * @cat Output
  * @method inspect
  * @param  {Object} obj An object or any other data item to be inspected.
- * @param  {Object} settings A settings object to control the function's behavior.
+ * @param  {Object} [settings] A settings object to control the function's behavior.
+ * @param  {Boolean} [settings.showProps] Show or hide properties. Default: <code>true</code>
+ * @param  {Boolean} [settings.showValues] Show or hide values. Default: <code>true</code>
+ * @param  {Boolean} [settings.showMethods] Show or hide methods. Default: <code>false</code>
+ * @param  {Number} [settings.maxLevel] How many levels of properties should be inspected recursively. Default: <code>1</code>
+ * @param  {Array} [settings.propList] Array of properties to show. Default: <code>[]</code> (no propList)
  *
  * @example
```

Comment:
the caption needs to be in the line of the `@example` tag. Should be

    * @example <caption>Inspecting a string</caption>